sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureClientRegistration.java

```diff
@@ -1,12 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.spring.aad.implementation;
+package com.azure.spring.aad.webapp;
 
 import org.springframework.security.oauth2.client.registration.ClientRegistration;
 
 import java.util.Set;
 
+/**
+ * Azure oauth2 client registration.
+ * It have 2 kind of scopes:
```

Comment:
Should be "It has", but it's not something big deal.

Reply:
Fixed. Thanks for point out.